### PR TITLE
Expose settings property on server object

### PR DESCRIPTION
**Pull request by smalot**
This is useful if you want to manipulate settings in a server listener for example to write a message when the server starts:

```Server listening on 8080```

### CODE REVIEW

File: src/Http/Server.php [hunk 1]
```diff
@@ -51,6 +51,16 @@ public function __construct(HttpConfiguration $settings = null)
         $this->settings = $settings;
     }
 
+    /**
+     * Return server configuration.
+     *
+     * @return HttpConfiguration
+     */
+    public function getSettings()
```
Comment (dkraczkowski) [inline src/Http/Server.php]: @smalot Missing return typehint.

**Comment by dkraczkowski** [comment]
@smalot Hello, thanks for your input. Could you please add missin return typehint?

**Comment by smalot** [comment]
return typehint added.

I made another update according to the issue I created.
If you prefer another MR I can split them

**Comment by smalot** [comment]
Travis job error seems not to be linked with my code ...

**Comment by dkraczkowski** [comment]
@smalot I have restarted travis job - seems OK now.

**Pull request merged**